modulesets-stable/gtk-osx-bootstrap.modules

```diff
@@ -80,7 +80,7 @@
   <!-- Cups is needed *only* for Tiger, and skipped otherwise -->
   <autotools id='cups' autogen-sh='configure' skip-autogen='never'
 	     autogenargs='DSOFLAGS="$LDFLAGS"'>
-    <branch module="cups/1.2.12/cups-1.2.12-source.tar.bz2"
+    <branch module="1.2.12/cups-1.2.12-source.tar.gz"
```

Comment:
No reason, i saw all other repos archived gz , so i just wanted to be consistent. Do you want me to resend fixed PR to your repository or here ?